pkg/cmd/roachtest/cluster.go

```diff
@@ -1002,7 +1003,8 @@ func attachToExistingCluster(
 			// If we're attaching to an existing cluster, we're not going to destroy it.
 			owned: false,
 		},
-		r: r,
+		localCertsDir: opt.certsDir,
```

Comment:
I wonder if operations should not follow the same logic as roachtest and copy the certs for itself to a temp directory. It can probably be done manually from the outside, but would have to be done each time the certs are updates. For normal storage clusters this is not a problem, to some degree. Growing a cluster / adding a load balancer does mutate the node cert. And with virtual clusters, there are new certs when a external process tenant is added.

Reply:
I think we're fine with one-time changes to node certs such as when we grow a cluster. We can always update the directory and restart the command. Do you expect the node certs to need fetching during the course of an operation?

Also good point on how `clusterImpl` sometimes overwrites the certs directory passed in. I see it only happens in `StartE` and `StartServiceForVirtualClusterE`. If we disallow both methods in operations, we could be fine with the current approach.